sdk/recoveryservices-backup/Microsoft.Azure.Management.RecoveryServices.Backup/src/recoveryservicesbackup/Generated/Models/AzureIaaSVMProtectionPolicy.cs

```diff
@@ -42,18 +42,24 @@ public AzureIaaSVMProtectionPolicy()
         /// backup policy.</param>
         /// <param name="retentionPolicy">Retention policy with the details on
         /// backup copy retention ranges.</param>
+        /// <param name="tieringPolicy">Tiering policy to automatically move
+        /// RPs to another tier
+        /// Key is Target Tier, defined in RecoveryPointTierType enum.
+        /// Tiering policy specifies the criteria to move RP to the target
+        /// tier.</param>
         /// <param name="instantRpRetentionRangeInDays">Instant RP retention
         /// policy range in days</param>
         /// <param name="timeZone">TimeZone optional input as string. For
         /// example: TimeZone = "Pacific Standard Time".</param>
         /// <param name="policyType">Possible values include: 'Invalid', 'V1',
         /// 'V2'</param>
-        public AzureIaaSVMProtectionPolicy(int? protectedItemsCount = default(int?), IList<string> resourceGuardOperationRequests = default(IList<string>), InstantRPAdditionalDetails instantRPDetails = default(InstantRPAdditionalDetails), SchedulePolicy schedulePolicy = default(SchedulePolicy), RetentionPolicy retentionPolicy = default(RetentionPolicy), int? instantRpRetentionRangeInDays = default(int?), string timeZone = default(string), string policyType = default(string))
+        public AzureIaaSVMProtectionPolicy(int? protectedItemsCount = default(int?), IList<string> resourceGuardOperationRequests = default(IList<string>), InstantRPAdditionalDetails instantRPDetails = default(InstantRPAdditionalDetails), SchedulePolicy schedulePolicy = default(SchedulePolicy), RetentionPolicy retentionPolicy = default(RetentionPolicy), IDictionary<string, TieringPolicy> tieringPolicy = default(IDictionary<string, TieringPolicy>), int? instantRpRetentionRangeInDays = default(int?), string timeZone = default(string), string policyType = default(string))
```

Comment:
breaking change

Reply:
Added customization files for breaking changes